main.go

```diff
@@ -126,6 +128,17 @@ func wrapMain() error {
 			Usage:  "Git tag",
 			EnvVar: "DRONE_TAG",
 		},
+		cli.StringSliceFlag{
+			Name:   "rollout_check",
+			Usage:  "List of deployments to check after apply (optional)",
+			EnvVar: "PLUGIN_ROLLOUT_CHECK",
+		},
+		cli.IntFlag{
+			Name:   "rollout_timeout",
```

Comment:
similarly, wait_seconds

might want to mention that exceeding the timeout fails the build